app/uk/gov/hmrc/digitalengagementplatformconfig/views/LandingView.scala.html

```diff
@@ -0,0 +1,23 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = "Hello from digital-engagement-platform-config", bodyClasses = None) {
+    <h1>@messages("landing.title")</h1>
+}
```

Comment:
Do we need to add:

        <script type="text/javascript" language="javascript" charset="utf-8" src="https://hmrc-uk.digital.nuance.com/chatskins/launch/inqChatLaunch10006719.js"> </script>

as per the existing page within lost-credentials-frontend service ?